packages/now-client/src/create-deployment.ts

```diff
@@ -53,6 +53,20 @@ export default function buildCreateDeployment(
 
     let rootFiles: string[];
 
+    if (Array.isArray(path)) {
+      for (const filePath of path) {
+        if (!isAbsolute(filePath)) {
+          yield { type: 'error', message: `Provided path ${filePath} is not absolute` };
+        }
+      }
+
+      return;
```

Comment:
I think this return is wrong because it will always prevent execution, even if all the paths are using an array.

Reply:
Indeed
Fixed